node/engine/policy/policy.go

```diff
@@ -0,0 +1,37 @@
+package policy
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/statechannels/go-nitro/internal/logging"
+	"github.com/statechannels/go-nitro/protocols"
+)
+
+// PolicyMaker is used to decide whether to approve or reject an objective
+type PolicyMaker interface {
+	ShouldApprove(o protocols.Objective) bool
+}
+
+type Policies struct {
+	policies []PolicyMaker
+}
+
+func NewPolicies(policies ...PolicyMaker) *Policies {
+	return &Policies{policies: policies}
+}
```

Comment:
Maybe doesn't bring much/any benefit, but wanted to suggest:
```suggestion
type Policies []PolicyMaker

func NewPolicies(policies ...PolicyMaker) *Policies {
	p := Policies(policies)
	return &p
}
```

Reply:
https://github.com/statechannels/go-nitro/pull/1599/commits/73192c47e719edf70493dce900e8b2ebed144d39